typing/jkind.ml

```diff
@@ -957,8 +960,18 @@ module Layout_and_axes = struct
                 Continue
                   { t with constr = Path.Map.add p (fuel - 1, args) constr }
               else Stop { t with fuel_status = Ran_out_of_fuel })
+          | Tvariant row -> (
+            let id = get_id (row_more row) in
```

Comment:
I think it is more customary to use `Btype.proxy` here instead of `row_more`.

Reply:
Done.